src/sql/insert_price.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO slippage_prices 
+(chain_name, source, block_number, tx_hash, token_address, price)
```

Comment:
Is there a file specifying the schema of the table? From the database it looks as if chain_name is just strings. For using it in prod, this should probably be changed to an enum.

Is the table with the new column created programmatically or manually?

Reply:
yes, chain_name is string. Once connected to the database, "\d table-name" shows the table schema. 
Cool, we can change it to enum. Table modifications are done manually. Do we want it programmatically?

Reply:
There _needs_ to be some documentation on how to set up the database. Easiest is to include a file containing the code for creating all the required tables.